bin/sdk/gutenberg.js

```diff
@@ -25,9 +26,10 @@ exports.compile = args => {
 		...{
 			context: __rootDir,
 			mode: options.mode,
-			entry: {
-				[ `${ name }-editor-script` ]: options.editorScript,
-			},
+			entry: pickBy( {
+				[ `${ name }-editor-script.js` ]: options.editorScript,
```

Comment:
Considering that editor script is required and the script will not continue if we omit that argument, do we need to pass it through `! isEmpty()` anyway?